src/cli/paths.rs

```diff
@@ -15,16 +16,23 @@ pub mod non_wasm {
         home_dir().join(Path::new(".lurk"))
     }
 
-    pub fn proofs_dir() -> PathBuf {
-        lurk_dir().join(Path::new("proofs"))
+    pub fn public_param_dir() -> PathBuf {
+        PUBLIC_PARAM_DIR
+            .get()
+            .unwrap()
+            .join(Path::new("public_params"))
     }
 
-    pub fn commits_dir() -> PathBuf {
-        lurk_dir().join(Path::new("commits"))
+    pub fn proof_dir() -> PathBuf {
+        PROOF_DIR.get().unwrap().join(Path::new("proofs"))
     }
 
-    pub fn lurk_leaf_dirs() -> [PathBuf; 2] {
-        [proofs_dir(), commits_dir()]
+    pub fn commit_dir() -> PathBuf {
+        COMMIT_DIR.get().unwrap().join(Path::new("commits"))
+    }
```

Comment:
Please use `expect` here and explain why it shouldn't panic at this point.

Also, let's use plurals: `public_params_dir`, `proofs_dir` and `commit_dir`